kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/RecordEncryptionFilter.java

```diff
@@ -157,4 +171,7 @@ private CompletionStage<PartitionData> maybeDecodeRecords(String topicName,
                 .thenApply(fpr::setRecords);
     }
 
+    private static boolean isEncryptionException(Throwable throwable) {
+        return EncryptionException.class.isAssignableFrom(throwable.getClass());
```

Comment:
Why not `return throwable instanceof EncryptionException;` ?

Reply:
No good answers, fixed in bc1e427fefacabfa03296c94462df9d22921a018